tools/nl/svindex_differ/README.md

```diff
@@ -2,14 +2,17 @@
 
 This is a command-line tool to validate SV index changes for regression.  It
 runs a bunch of golden queries (in [queryset.csv](queryset.csv)) against a
-`test` index and a `base` index, and reports diffs in an HTML report.
+`test` index and a `base` index, and reports diffs in an HTML report that is saved in `gs://datcom-embedding-diffs`.
```

Comment:
nit: maybe link to https://pantheon.corp.google.com/storage/browser/datcom-embedding-diffs